packages/landing/src/components/layer.switcher.dropdown.tsx

```diff
@@ -17,6 +17,9 @@ export interface LayerSwitcherDropdownState {
   zoomToExtent: boolean;
   currentLayer: string;
 }
+
+const ignoredLayers = ['all'];
```

Comment:
`new Set(['all'])` granted it doenst matter where `n=1` but if this ever gets to be bigger than 5 or so items it could become problematic.